READMEs/parameters.md

```diff
@@ -38,8 +38,6 @@ d = {
 ocean = Ocean(d)
 ```
 
-For legacy support, you can also use `metadataStoreUri` instead of `metadataCacheUri`.
```

Comment:
this is still true, I think. We name the variable metadata_cache_uri everywhere, but the user can configure it using any of these keys.

Reply:
I have talked with @DMats who wasn't sure about the fallback that's why I removed it, but I can add it right now.

Reply:
Apologies, I must have mispoke. Calina's right.